bin/assigner/src/main.cpp

```diff
@@ -530,7 +536,8 @@ int main(int argc, char *argv[]) {
             ("stack-size,s", boost::program_options::value<long>(), "Stack size in bytes")
             ("check", "Check satisfiability of the generated circuit")
             ("log-level,l", boost::program_options::value<std::string>(), "Log level (trace, debug, info, warning, error, fatal)")
-            ("print_circuit_output", "print output of the circuit")
+            ("print_circuit_output", "deprecated, use \"-o print_circuit_output\" instead")
+            ("print-circuit-output,o", boost::program_options::value<std::string>(), "Native elliptic curve type (pallas, vesta, ed25519, bls12381)")
```

Comment:
Duplicated option + copy-pasted description. I think you should check the documentation of the parser. And please do not use `-o`, it could be confusing because it's the commonly used option for an output file.